packages/pdf-generator/src/components/CSAwIcon.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Canvas } from '@react-pdf/renderer';
+
+const oval = `M 12.00,6.00
+           C 12.00,9.31 9.31,12.00 6.00,12.00
+             2.69,12.00 0.00,9.31 0.00,6.00
+             0.00,2.69 2.69,0.00 6.00,0.00
+             6.01,-0.01 11.71,0.25 12.00,6.00 Z
+           M 56.25,20.75M 29.00,10.00`;
+
+const shape = `M 10.15,5.57 C 9.36,4.04 7.79,3.00 6.00,3.00
+             4.21,3.00 2.64,4.04 1.85,5.57
+             1.78,5.70 1.78,5.86 1.85,5.99
+             2.64,7.53 4.21,8.56 6.00,8.56
+             7.79,8.56 9.36,7.53 10.15,5.99
+             10.22,5.86 10.22,5.70 10.15,5.57 Z
+           M 6.00,7.87
+           C 4.84,7.87 3.90,6.93 3.90,5.78
+             3.90,4.63 4.84,3.70 6.00,3.70
+             7.16,3.70 8.10,4.63 8.10,5.78
+             8.10,6.34 7.88,6.87 7.49,7.26
+             7.09,7.65 6.56,7.87 6.00,7.87
+             6.00,7.87 6.00,7.87 6.00,7.87 Z
+           M 6.00,4.39
+           C 5.88,4.39 5.75,4.41 5.63,4.45
+             5.84,4.72 5.81,5.10 5.56,5.35
+             5.32,5.59 4.93,5.62 4.66,5.42
+             4.50,5.99 4.73,6.60 5.23,6.93
+             5.72,7.26 6.38,7.24 6.85,6.88
+             7.33,6.52 7.52,5.89 7.33,5.33
+             7.13,4.77 6.60,4.39 6.00,4.39
+             6.00,4.39 6.00,4.39 6.00,4.39 Z`;
+
+const path = `M 6.00,7.87
+           C 4.84,7.87 3.90,6.93 3.90,5.78
+             3.90,4.63 4.84,3.70 6.00,3.70
+             7.16,3.70 8.10,4.63 8.10,5.78
+             8.10,6.34 7.88,6.87 7.49,7.26
+             7.09,7.65 6.56,7.87 6.00,7.87
+             6.00,7.87 6.00,7.87 6.00,7.87 Z`;
+
+const CSAwIcon = ({ style, ...props }) => (
+    <Canvas
+        {...props}
+        style={{
+            ...style,
+            width: 16,
```

Comment:
I'd put these values to `utils/styles.js` file so we don't have to hunt down where does these values come from.